File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.py
def solution(my_string):
    answer = [0]*52
    all_alpha = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
    for i in my_string:
        answer[all_alpha.find(i)] += 1
    return answer

File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.js
function solution(my_string) {
    let answer = Array(52).fill(0);
    let all_alpha = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
    for (let i = 0; i <= my_string.length; i++){
        let index = all_alpha.indexOf(my_string[i])
        if (index !== -1) answer[index]++;
    }
    return answer;
}